python/cudf/cudf/io/parquet.py

```diff
@@ -203,6 +205,48 @@ def read_parquet(
             )
         filepaths_or_buffers.append(tmp_source)
 
+    if filters is not None:
+        # Convert filters to ds.Expression
+        filters = pq._filters_to_expression(filters)
```

Comment:
I'm guessing there's no public API that we could use here instead? Could we open a JIRA issue against Arrow to make this a public API method in the future?

Reply:
I opened [an issue](https://issues.apache.org/jira/browse/ARROW-9672).

For now, because `filters_to_expression` is small and contained, I just copied over the 2 relevant functions as private helper functions in this file `parquet.py`. Let me know if that is fine.

Reply:
Can you add a comment of a `#TODO` and link to that JIRA issue?